docs/DeveloperGuide.md

```diff
@@ -90,29 +90,29 @@ Here's a (partial) class diagram of the `Logic` component:
 
 <puml src="diagrams/LogicClassDiagram.puml" width="550"/>
 
-The sequence diagram below illustrates the interactions within the `Logic` component, taking `execute("delete 1")` API call as an example.
+The sequence diagram below illustrates the interactions within the `Logic` component, taking `execute("rmt 1")` API call as an example.
```

Comment:
maybe can add what rmt 1 means

Reply:
you did not add what rmt 1 means

Reply:
Is line 94 the description for command 'rmt 1' not clear?